### ISSUE TITLE
Device folders showing up for backup, but no albums

### ISSUE THREAD

**Issue by Sander-Brilman** [immich-app/immich#9816]
### The bug

When i try to backup an album in immich it does not show all albums on my device. Immich only shows the albums circled in red (albums like camera, whatapp images, ect..). which are somehow a different type of album then those i created manually (circled in blue).

the albums circled in red i cannot manually create or add images to.

![modified](https://github.com/immich-app/immich/assets/99390032/8a22eba8-e19e-479b-8076-612237997f06)

Both the "id" album and the "immich backup" album which i manually created dont show up in the list for backup
![Screenshot_20240528-100842](https://github.com/immich-app/immich/assets/99390032/51286464-17e0-4093-afab-da0f3112fb85)
![Screenshot_20240528-100838](https://github.com/immich-app/immich/assets/99390032/208a510d-1d45-45c2-93eb-38fbeecd7445)

This is some info about the app im running. I downloaded the app via the playstore on my google pixel 6a

![image0](https://github.com/immich-app/immich/assets/99390032/af685f58-0f00-40e9-9561-d2745bd751f1)

Is there something i did wrong or is this a bug?
Let me know if you need more information

### The OS that Immich Server is running on

Android 14

### Version of Immich Server

1.105.1

### Version of Immich Mobile App

1.105.0 build 139

### Platform with the issue

- [ ] Server
- [ ] Web
- [X] Mobile

### Your docker-compose.yml content

```YAML
n/a
```

### Your .env content

```Shell
n/a
```

### Reproduction steps

```bash
1. download the immich app from the google play store
2. login into your server
3. select album for backup
```

### Relevant log output

```shell
2024-05-28 10:21:54.839713 | INFO     | BackupNotifier       | No assets are selected for back up |
2024-05-28 10:21:43.912408 | INFO     | BackupNotifier       | No assets are selected for back up |
2024-05-28 10:21:43.859027 | INFO     | BackupNotifier       | _getBackupAlbumsInfo: Found 10 available albums |
2024-05-28 10:21:43.850905 | INFO     | BackupNotifier       | Found 10 local albums |
2024-05-28 10:21:38.607808 | INFO     | BackupNotifier       | No assets are selected for back up |
2024-05-28 10:21:38.550523 | INFO     | BackupNotifier       | _getBackupAlbumsInfo: Found 10 available albums |
2024-05-28 10:21:38.548828 | INFO     | BackupNotifier       | Found 10 local albums |
2024-05-28 10:21:36.946186 | INFO     | AssetNotifier        | Load assets: 171ms |
2024-05-28 10:19:33.863044 | INFO     | BackupNotifier       | No assets are selected for back up |
2024-05-28 10:19:33.816690 | INFO     | BackupNotifier       | _getBackupAlbumsInfo: Found 10 available albums |
2024-05-28 10:19:33.813644 | INFO     | BackupNotifier       | Found 10 local albums |
2024-05-28 10:19:31.923999 | INFO     | AssetNotifier        | Load assets: 126ms |
2024-05-28 10:19:28.044509 | INFO     | BackupNotifier       | No assets are selected for back up |
2024-05-28 10:19:27.260545 | INFO     | AssetNotifier        | Load assets: 191ms |
2024-05-28 10:08:32.184860 | INFO     | BackupNotifier       | No assets are selected for back up |
2024-05-28 10:08:32.120424 | INFO     | BackupNotifier       | _getBackupAlbumsInfo: Found 10 available albums |
2024-05-28 10:08:32.108230 | INFO     | BackupNotifier       | Found 10 local albums |
2024-05-28 10:08:31.548988 | INFO     | BackupNotifier       | No assets are selected for back up |
2024-05-28 10:08:31.255769 | INFO     | BackupNotifier       | _getBackupAlbumsInfo: Found 10 available albums |
2024-05-28 10:08:31.254329 | INFO     | BackupNotifier       | Found 10 local albums |
2024-05-28 10:08:28.142942 | INFO     | AssetNotifier        | Load assets: 157ms |
2024-05-28 09:45:46.543179 | INFO     | AssetNotifier        | Load assets: 166ms |
2024-05-28 09:44:48.055003 | INFO     | BackupNotifier       | No assets are selected for back up |
2024-05-28 09:44:47.369175 | INFO     | AssetNotifier        | Load assets: 117ms |
2024-05-28 09:43:50.252611 | INFO     | BackupNotifier       | No assets are selected for back up |
2024-05-28 09:43:50.201864 | INFO     | BackupNotifier       | _getBackupAlbumsInfo: Found 10 available albums |
2024-05-28 09:43:50.200103 | INFO     | BackupNotifier       | Found 10 local albums |
2024-05-28 09:43:49.417617 | INFO     | BackupNotifier       | No assets are selected for back up |
2024-05-28 09:43:47.602381 | INFO     | AssetNotifier        | Load assets: 138ms |
2024-05-28 09:39:52.468592 | INFO     | BackupNotifier       | No assets are selected for back up |
2024-05-28 09:39:52.390557 | INFO     | BackupNotifier       | _getBackupAlbumsInfo: Found 10 available albums |
2024-05-28 09:39:52.370557 | INFO     | BackupNotifier       | Found 10 local albums |
2024-05-28 09:39:51.676629 | INFO     | BackupNotifier       | No assets are selected for back up |
2024-05-28 09:39:51.609733 | INFO     | BackupNotifier       | _getBackupAlbumsInfo: Found 10 available albums |
2024-05-28 09:39:51.602961 | INFO     | BackupNotifier       | Found 10 local albums |
2024-05-28 09:39:48.554089 | INFO     | BackupNotifier       | No assets are selected for back up |
2024-05-28 09:39:45.524290 | INFO     | BackupNotifier       | _getBackupAlbumsInfo: Found 10 available albums |
2024-05-28 09:39:45.508357 | INFO     | BackupNotifier       | Found 10 local albums |
2024-05-28 09:39:44.316240 | INFO     | BackupNotifier       | _getBackupAlbumsInfo: Found 10 available albums |
2024-05-28 09:39:44.315417 | INFO     | BackupNotifier       | Found 10 local albums |
2024-05-28 09:38:36.919483 | INFO     | AlbumService         | Syncing completed. Changes: false |
2024-05-28 09:38:36.913303 | INFO     | AlbumService         | 'Recents' is not selected, keeping only selected albums |
2024-05-28 09:38:36.913161 | INFO     | AlbumService         | Found 10 device albums |
2024-05-28 09:38:33.266965 | INFO     | AlbumService         | Syncing completed. Changes: false |
2024-05-28 09:38:33.255395 | INFO     | AlbumService         | 'Recents' is not selected, keeping only selected albums |
2024-05-28 09:38:33.255224 | INFO     | AlbumService         | Found 10 device albums |
2024-05-28 10:19:33.863044 | INFO     | BackupNotifier       | No assets are selected for back up |
2024-05-28 10:19:33.816690 | INFO     | BackupNotifier       | _getBackupAlbumsInfo: Found 10 available albums |
2024-05-28 10:19:33.813644 | INFO     | BackupNotifier       | Found 10 local albums |
2024-05-28 10:19:31.923999 | INFO     | AssetNotifier        | Load assets: 126ms |
2024-05-28 10:19:28.044509 | INFO     | BackupNotifier       | No assets are selected for back up |
2024-05-28 10:19:27.260545 | INFO     | AssetNotifier        | Load assets: 191ms |
2024-05-28 10:08:32.184860 | INFO     | BackupNotifier       | No assets are selected for back up |
2024-05-28 10:08:32.120424 | INFO     | BackupNotifier       | _getBackupAlbumsInfo: Found 10 available albums |
2024-05-28 10:08:32.108230 | INFO     | BackupNotifier       | Found 10 local albums |
2024-05-28 10:08:31.548988 | INFO     | BackupNotifier       | No assets are selected for back up |
2024-05-28 10:08:31.255769 | INFO     | BackupNotifier       | _getBackupAlbumsInfo: Found 10 available albums |
2024-05-28 10:08:31.254329 | INFO     | BackupNotifier       | Found 10 local albums |
2024-05-28 10:08:28.142942 | INFO     | AssetNotifier        | Load assets: 157ms |
2024-05-28 09:45:46.543179 | INFO     | AssetNotifier        | Load assets: 166ms |
2024-05-28 09:44:48.055003 | INFO     | BackupNotifier       | No assets are selected for back up |
2024-05-28 09:44:47.369175 | INFO     | AssetNotifier        | Load assets: 117ms |
2024-05-28 09:43:50.252611 | INFO     | BackupNotifier       | No assets are selected for back up |
2024-05-28 09:43:50.201864 | INFO     | BackupNotifier       | _getBackupAlbumsInfo: Found 10 available albums |
2024-05-28 09:43:50.200103 | INFO     | BackupNotifier       | Found 10 local albums |
2024-05-28 09:43:49.417617 | INFO     | BackupNotifier       | No assets are selected for back up |
2024-05-28 09:43:47.602381 | INFO     | AssetNotifier        | Load assets: 138ms |
2024-05-28 09:39:52.468592 | INFO     | BackupNotifier       | No assets are selected for back up |
2024-05-28 09:39:52.390557 | INFO     | BackupNotifier       | _getBackupAlbumsInfo: Found 10 available albums |
2024-05-28 09:39:52.370557 | INFO     | BackupNotifier       | Found 10 local albums |
2024-05-28 09:39:51.676629 | INFO     | BackupNotifier       | No assets are selected for back up |
2024-05-28 09:39:51.609733 | INFO     | BackupNotifier       | _getBackupAlbumsInfo: Found 10 available albums |
2024-05-28 09:39:51.602961 | INFO     | BackupNotifier       | Found 10 local albums |
2024-05-28 09:39:48.554089 | INFO     | BackupNotifier       | No assets are selected for back up |
2024-05-28 09:39:45.524290 | INFO     | BackupNotifier       | _getBackupAlbumsInfo: Found 10 available albums |
2024-05-28 09:39:45.508357 | INFO     | BackupNotifier       | Found 10 local albums |
2024-05-28 09:39:44.316240 | INFO     | BackupNotifier       | _getBackupAlbumsInfo: Found 10 available albums |
2024-05-28 09:39:44.315417 | INFO     | BackupNotifier       | Found 10 local albums |
2024-05-28 09:38:36.919483 | INFO     | AlbumService         | Syncing completed. Changes: false |
2024-05-28 09:38:36.913303 | INFO     | AlbumService         | 'Recents' is not selected, keeping only selected albums |
2024-05-28 09:38:36.913161 | INFO     | AlbumService         | Found 10 device albums |
2024-05-28 09:38:33.266965 | INFO     | AlbumService         | Syncing completed. Changes: false |
2024-05-28 09:38:33.255395 | INFO     | AlbumService         | 'Recents' is not selected, keeping only selected albums |
2024-05-28 09:38:33.255224 | INFO     | AlbumService         | Found 10 device albums |
2024-05-28 09:36:45.662422 | INFO     | BackupNotifier       | _getBackupAlbumsInfo: Found 10 available albums |
2024-05-28 09:36:45.661185 | INFO     | BackupNotifier       | Found 10 local albums |
2024-05-28 09:36:41.214214 | INFO     | AlbumService         | Syncing completed. Changes: true |
2024-05-28 09:36:41.213750 | INFO     | SyncService          | Added a new local album to DB: Pictures |
2024-05-28 09:36:41.173570 | INFO     | SyncService          | Upserted 20 assets into the DB |
2024-05-28 09:36:41.170245 | INFO     | SyncService          | 0 assets already existed in DB, to upsert 20 |
2024-05-28 09:36:41.135792 | INFO     | SyncService          | Syncing a new local album to DB: Pictures |
2024-05-28 09:36:41.134799 | INFO     | AlbumService         | 'Recents' is not selected, keeping only selected albums |
2024-05-28 09:36:41.134478 | INFO     | AlbumService         | Found 10 device albums |
2024-05-28 09:36:35.757602 | INFO     | BackupNotifier       | No assets are selected for back up |
2024-05-28 09:36:35.688611 | INFO     | BackupNotifier       | _getBackupAlbumsInfo: Found 10 available albums |
2024-05-28 09:36:35.676930 | INFO     | BackupNotifier       | Found 10 local albums |
2024-05-28 09:36:22.089733 | INFO     | BackupNotifier       | No assets are selected for back up |
2024-05-28 09:36:22.013555 | INFO     | BackupNotifier       | _getBackupAlbumsInfo: Found 10 available albums |
2024-05-28 09:36:22.009870 | INFO     | BackupNotifier       | Found 10 local albums |
2024-05-28 09:36:04.909113 | INFO     | BackupNotifier       | No assets are selected for back up |
2024-05-28 09:36:04.842753 | INFO     | BackupNotifier       | _getBackupAlbumsInfo: Found 10 available albums |
2024-05-28 09:36:04.840295 | INFO     | BackupNotifier       | Found 10 local albums |
2024-05-28 09:36:00.456970 | INFO     | BackupNotifier       | No assets are selected for back up |
2024-05-28 09:35:55.934913 | INFO     | BackupNotifier       | No assets are selected for back up |
2024-05-28 09:35:55.580720 | INFO     | BackupNotifier       | _getBackupAlbumsInfo: Found 10 available albums |
2024-05-28 09:35:55.578909 | INFO     | BackupNotifier       | Found 10 local albums |
2024-05-28 09:35:52.738167 | INFO     | BackupNotifier       | No assets are selected for back up |
2024-05-28 09:35:17.116671 | INFO     | BackupNotifier       | No assets are selected for back up |
2024-05-28 09:35:17.056232 | INFO     | BackupNotifier       | _getBackupAlbumsInfo: Found 10 available albums |
2024-05-28 09:35:17.055278 | INFO     | BackupNotifier       | Found 10 local albums |
2024-05-28 09:35:16.200540 | INFO     | AssetNotifier        | Load assets: 174ms |
2024-05-28 09:33:16.133283 | INFO     | BackupNotifier       | No assets are selected for back up |
2024-05-28 09:33:03.030606 | INFO     | AssetNotifier        | Load assets: 390ms |
2024-05-28 09:22:50.271404 | INFO     | BackupNotifier       | No assets are selected for back up |
2024-05-28 09:22:50.183576 | INFO     | BackupNotifier       | _getBackupAlbumsInfo: Found 10 available albums |
2024-05-28 09:22:50.183012 | INFO     | BackupNotifier       | Found 10 local albums |
2024-05-28 09:22:48.927275 | INFO     | BackupNotifier       | No assets are selected for back up |
2024-05-28 09:22:48.871326 | INFO     | BackupNotifier       | _getBackupAlbumsInfo: Found 10 available albums |
2024-05-28 09:22:48.869695 | INFO     | BackupNotifier       | Found 10 local albums |
2024-05-28 09:22:47.644499 | INFO     | AssetNotifier        | Load assets: 167ms |
2024-05-28 09:22:47.359017 | INFO     | ApiService           | Pinging server with response code 200 |
2024-05-28 09:07:47.683119 | INFO     | BackupNotifier       | No assets are selected for back up |
2024-05-28 09:07:27.526056 | INFO     | AssetNotifier        | Load assets: 192ms |
2024-05-28 09:01:02.207240 | INFO     | BackupNotifier       | No assets are selected for back up |
2024-05-28 09:01:02.143702 | INFO     | BackupNotifier       | _getBackupAlbumsInfo: Found 11 available albums |
2024-05-28 09:01:02.142632 | INFO     | BackupNotifier       | Found 11 local albums |
2024-05-28 09:00:55.095060 | INFO     | BackupNotifier       | No assets are selected for back up |
2024-05-28 09:00:55.062151 | INFO     | BackupNotifier       | _getBackupAlbumsInfo: Found 11 available albums |
2024-05-28 09:00:55.060065 | INFO     | BackupNotifier       | Found 11 local albums |
2024-05-28 09:00:51.739952 | INFO     | AssetNotifier        | Load assets: 216ms |
2024-05-28 08:59:04.584485 | INFO     | BackupNotifier       | No assets are selected for back up |
2024-05-28 08:58:51.230543 | INFO     | AssetNotifier        | Load assets: 187ms |
2024-05-28 08:57:04.616185 | INFO     | BackupNotifier       | No assets are selected for back up |
2024-05-28 08:57:04.568757 | INFO     | BackupNotifier       | _getBackupAlbumsInfo: Found 11 available albums |
2024-05-28 08:57:04.556064 | INFO     | BackupNotifier       | Found 11 local albums |
2024-05-28 08:57:03.178974 | INFO     | BackupNotifier       | No assets are selected for back up |
2024-05-28 08:57:03.120941 | INFO     | BackupNotifier       | _getBackupAlbumsInfo: Found 11 available albums |
2024-05-28 08:57:03.120085 | INFO     | BackupNotifier       | Found 11 local albums |
2024-05-28 08:57:00.946346 | INFO     | AssetNotifier        | Load assets: 176ms |
2024-05-28 08:57:00.678744 | INFO     | ApiService           | Pinging server with response code 200 |
2024-05-28 08:56:49.412650 | INFO     | AssetNotifier        | Load assets: 164ms |
2024-05-28 08:56:49.173914 | INFO     | ApiService           | Pinging server with response code 200 |
2024-05-28 08:56:09.835580 | INFO     | BackupNotifier       | No assets are selected for back up |
2024-05-28 08:56:09.777590 | INFO     | BackupNotifier       | _getBackupAlbumsInfo: Found 11 available albums |
2024-05-28 08:56:09.776680 | INFO     | BackupNotifier       | Found 11 local albums |
2024-05-28 08:56:08.976373 | INFO     | BackupNotifier       | No assets are selected for back up |
2024-05-28 08:56:08.936010 | INFO     | BackupNotifier       | _getBackupAlbumsInfo: Found 11 available albums |
2024-05-28 08:56:08.935154 | INFO     | BackupNotifier       | Found 11 local albums |
2024-05-28 08:56:06.010652 | INFO     | AssetNotifier        | Load assets: 181ms |
2024-05-28 08:56:05.678607 | INFO     | ApiService           | Pinging server with response code 200 |
2024-05-28 08:55:57.535033 | INFO     | BackupNotifier       | No assets are selected for back up |
2024-05-28 08:55:57.457198 | INFO     | BackupNotifier       | _getBackupAlbumsInfo: Found 11 available albums |
2024-05-28 08:55:57.441071 | INFO     | BackupNotifier       | Found 11 local albums |
2024-05-28 08:55:56.454573 | INFO     | BackupNotifier       | No assets are selected for back up |
2024-05-28 08:55:56.379730 | INFO     | BackupNotifier       | _getBackupAlbumsInfo: Found 11 available albums |
2024-05-28 08:55:56.373315 | INFO     | BackupNotifier       | Found 11 local albums |
2024-05-28 08:55:30.792707 | INFO     | BackupNotifier       | No assets are selected for back up |
2024-05-28 08:55:23.543217 | INFO     | BackupNotifier       | No assets are selected for back up |
2024-05-28 08:55:23.481012 | INFO     | BackupNotifier       | _getBackupAlbumsInfo: Found 11 available albums |
2024-05-28 08:55:23.479875 | INFO     | BackupNotifier       | Found 11 local albums |
2024-05-28 08:55:23.410409 | INFO     | AssetNotifier        | Load assets: 31791ms |
2024-05-28 08:55:23.296851 | WARNING  | UserService          | Failed to refresh users |
2024-05-28 08:55:23.199692 | WARNING  | UserService          | Failed get all users | ApiException 400: Socket operation failed: GET /user (Inner exception: Software caused connection abort)

#0      IOClient.send (package:http/src/io_client.dart:94)
<asynchronous suspension>
#1      BaseClient._sendUnstreamed (package:http/src/base_client.dart:93)
<asynchronous suspension>
#2      ApiClient.invokeAPI (package:openapi/api_client.dart:101)
<asynchronous suspension>
#3      UserApi.getAllUsers (package:openapi/api/user_api.dart:243)
<asynchronous suspension>
#4      UserService._getAllUsers (package:immich_mobile/services/user.service.dart:42)
<asynchronous suspension>
#5      UserService.getUsersFromServer (package:immich_mobile/services/user.service.dart:74)
<asynchronous suspension>
#6      UserService.refreshUsers (package:immich_mobile/services/user.service.dart:115)
<asynchronous suspension>
#7      AssetNotifier.getAllAsset (package:immich_mobile/providers/asset.provider.dart:53)
<asynchronous suspension>
 |
2024-05-28 08:55:22.753109 | INFO     | BackupNotifier       | No assets are selected for back up |
2024-05-28 08:55:22.663236 | INFO     | BackupNotifier       | _getBackupAlbumsInfo: Found 11 available albums |
2024-05-28 08:55:22.660805 | INFO     | BackupNotifier       | Found 11 local albums |
2024-05-28 08:48:32.366257 | INFO     | AssetNotifier        | Load assets: 151ms |
2024-05-28 08:42:19.600301 | INFO     | AssetNotifier        | Load assets: 159ms |
2024-05-28 08:42:16.510715 | INFO     | BackupNotifier       | No assets are selected for back up |
2024-05-28 08:41:25.148599 | INFO     | BackupNotifier       | No assets are selected for back up |
2024-05-28 08:41:25.086729 | INFO     | BackupNotifier       | _getBackupAlbumsInfo: Found 11 available albums |
2024-05-28 08:41:25.084878 | INFO     | BackupNotifier       | Found 11 local albums |
2024-05-28 08:41:24.183305 | INFO     | BackupNotifier       | No assets are selected for back up |
2024-05-28 08:41:24.115934 | INFO     | BackupNotifier       | _getBackupAlbumsInfo: Found 11 available albums |
2024-05-28 08:41:24.114319 | INFO     | BackupNotifier       | Found 11 local albums |
2024-05-28 08:41:14.373520 | INFO     | BackupNotifier       | No assets are selected for back up |
2024-05-28 08:41:14.336478 | INFO     | BackupNotifier       | _getBackupAlbumsInfo: Found 11 available albums |
2024-05-28 08:41:14.327208 | INFO     | BackupNotifier       | Found 11 local albums |
2024-05-28 08:41:08.104118 | INFO     | AssetNotifier        | Load assets: 395ms |
2024-05-28 08:41:07.551692 | INFO     | ApiService           | Pinging server with response code 200 |
2024-05-28 08:40:20.837181 | INFO     | BackupNotifier       | No assets are selected for back up |
2024-05-28 08:40:20.758543 | INFO     | BackupNotifier       | _getBackupAlbumsInfo: Found 11 available albums |
2024-05-28 08:40:20.757137 | INFO     | BackupNotifier       | Found 11 local albums |
2024-05-28 08:40:17.242042 | INFO     | AssetNotifier        | Load assets: 229ms |
2024-05-28 08:40:17.238692 | INFO     | SyncService          | Upserted 110 assets into the DB |
2024-05-28 08:40:16.932253 | INFO     | ApiService           | Pinging server with response code 200 |
2024-05-28 08:39:15.671262 | INFO     | AssetNotifier        | Load assets: 183ms |
2024-05-28 08:39:06.885858 | INFO     | AssetNotifier        | Load assets: 175ms |
2024-05-28 08:38:42.201310 | INFO     | BackupNotifier       | No assets are selected for back up |
2024-05-28 08:38:42.126690 | INFO     | BackupNotifier       | _getBackupAlbumsInfo: Found 11 available albums |
2024-05-28 08:38:42.120791 | INFO     | BackupNotifier       | Found 11 local albums |
2024-05-28 08:37:28.215085 | INFO     | BackupNotifier       | No assets are selected for back up |
2024-05-28 08:37:19.924941 | INFO     | BackupNotifier       | _getBackupAlbumsInfo: Found 11 available albums |
2024-05-28 08:37:19.922809 | INFO     | BackupNotifier       | Found 11 local albums |
2024-05-28 08:37:14.980216 | INFO     | BackupNotifier       | _getBackupAlbumsInfo: Found 11 available albums |
2024-05-28 08:37:14.969444 | INFO     | BackupNotifier       | Found 11 local albums |
2024-05-28 08:37:14.167820 | INFO     | AssetNotifier        | Load assets: 531ms |
2024-05-28 08:37:14.167687 | INFO     | AlbumService         | Syncing completed. Changes: false |
2024-05-28 08:37:14.153619 | INFO     | AlbumService         | 'Recents' is not selected, keeping only selected albums |
2024-05-28 08:37:14.153511 | INFO     | AlbumService         | Found 11 device albums |
2024-05-28 08:37:14.050718 | INFO     | SyncService          | Upserted 17 assets into the DB |
2024-05-28 08:37:13.949192 | INFO     | AlbumService         | Syncing completed. Changes: false |
2024-05-28 08:37:13.746878 | INFO     | AlbumService         | 'Recents' is not selected, keeping only selected albums |
2024-05-28 08:37:13.746806 | INFO     | AlbumService         | Found 11 device albums |
2024-05-28 08:37:13.536314 | INFO     | ApiService           | Pinging server with response code 200 |
2024-05-28 08:34:00.832641 | INFO     | AssetNotifier        | Load assets: 539ms |
2024-05-28 08:34:00.832421 | INFO     | AlbumService         | Syncing completed. Changes: false |
2024-05-28 08:34:00.763533 | INFO     | AlbumService         | 'Recents' is not selected, keeping only selected albums |
2024-05-28 08:34:00.763427 | INFO     | AlbumService         | Found 11 device albums |
2024-05-28 08:34:00.651212 | INFO     | SyncService          | Upserted 28 assets into the DB |
2024-05-28 08:33:20.462193 | INFO     | BackupNotifier       | _getBackupAlbumsInfo: Found 11 available albums |
2024-05-28 08:33:20.459730 | INFO     | BackupNotifier       | Found 11 local albums |
2024-05-28 08:33:19.070581 | INFO     | AlbumService         | Syncing completed. Changes: true |
2024-05-28 08:33:19.015294 | INFO     | SyncService          | Added a new local album to DB: Pictures |
2024-05-28 08:33:19.010203 | INFO     | SyncService          | Upserted 20 assets into the DB |
2024-05-28 08:33:19.008257 | INFO     | SyncService          | 0 assets already existed in DB, to upsert 20 |
2024-05-28 08:33:18.430411 | INFO     | SyncService          | Syncing a new local album to DB: Pictures |
2024-05-28 08:33:18.429143 | INFO     | AlbumService         | 'Recents' is not selected, keeping only selected albums |
2024-05-28 08:33:18.428952 | INFO     | AlbumService         | Found 11 device albums |
2024-05-28 08:33:09.254332 | INFO     | BackupNotifier       | _getBackupAlbumsInfo: Found 11 available albums |
2024-05-28 08:33:09.253040 | INFO     | BackupNotifier       | Found 11 local albums |
2024-05-28 08:33:07.852226 | INFO     | BackupNotifier       | _getBackupAlbumsInfo: Found 11 available albums |
2024-05-28 08:33:07.850707 | INFO     | BackupNotifier       | Found 11 local albums |
2024-05-28 08:33:02.339217 | INFO     | AssetNotifier        | Load assets: 1497ms |
2024-05-28 08:33:02.339147 | INFO     | AlbumService         | Syncing completed. Changes: true |
2024-05-28 08:33:02.339017 | INFO     | SyncService          | Synced changes of local album Camera to DB |
2024-05-28 08:33:01.771973 | INFO     | AlbumService         | 'Recents' is not selected, keeping only selected albums |
2024-05-28 08:33:01.771876 | INFO     | AlbumService         | Found 11 device albums |
2024-05-28 08:33:01.530353 | INFO     | SyncService          | Upserted 82 assets into the DB |
2024-05-27 22:19:56.119274 | INFO     | AlbumService         | Syncing completed. Changes: true |
2024-05-27 22:19:56.119094 | INFO     | SyncService          | Added a new local album to DB: Camera |
2024-05-27 22:19:56.035679 | INFO     | SyncService          | Upserted 274 assets into the DB |
2024-05-27 22:19:56.005745 | INFO     | SyncService          | 0 assets already existed in DB, to upsert 274 |
2024-05-27 22:19:55.768640 | INFO     | BackupNotifier       | _getBackupAlbumsInfo: Found 11 available albums |
2024-05-27 22:19:55.767305 | INFO     | BackupNotifier       | Found 11 local albums |
2024-05-27 22:19:53.293259 | INFO     | AlbumService         | refreshDeviceAlbums is already in progress |
2024-05-27 22:19:46.743406 | INFO     | BackupNotifier       | _getBackupAlbumsInfo: Found 11 available albums |
2024-05-27 22:19:46.742106 | INFO     | BackupNotifier       | Found 11 local albums |
2024-05-27 22:19:44.499488 | INFO     | SyncService          | Syncing a new local album to DB: Camera |
2024-05-27 22:19:44.498652 | INFO     | AlbumService         | 'Recents' is not selected, keeping only selected albums |
2024-05-27 22:19:44.498485 | INFO     | AlbumService         | Found 11 device albums |
2024-05-27 22:19:27.304066 | INFO     | BackupNotifier       | No assets are selected for back up |
2024-05-27 22:19:27.246802 | INFO     | BackupNotifier       | _getBackupAlbumsInfo: Found 11 available albums |
2024-05-27 22:19:27.245983 | INFO     | BackupNotifier       | Found 11 local albums |
2024-05-27 22:19:21.693284 | INFO     | BackupNotifier       | No assets are selected for back up |
2024-05-27 22:19:21.645429 | INFO     | BackupNotifier       | _getBackupAlbumsInfo: Found 11 available albums |
2024-05-27 22:19:21.642626 | INFO     | BackupNotifier       | Found 11 local albums |
2024-05-27 22:19:08.809712 | INFO     | AssetNotifier        | Load assets: 283ms |
2024-05-27 22:17:43.096772 | INFO     | AssetNotifier        | Load assets: 172ms |
2024-05-27 22:16:58.647200 | INFO     | AssetNotifier        | Load assets: 213ms |
2024-05-27 22:16:58.040349 | INFO     | ApiService           | Pinging server with response code 200 |
2024-05-27 22:16:10.363593 | INFO     | ApiService           | Pinging server with response code 200 |
```

**Comment by alextran1502** [comment]
Well looks like the Immich Backup and Id album is on Google Photos, not on your device 😅

**Comment by Sander-Brilman** [comment]
i thought they were one and the same, guess i am wrong about that. 

Does that mean i need a 3'd party app to create device-local albums that i can then upload?
Or is there maybe a plugin that gives immich access to google pixel?

**Comment by aviv926** [comment]
> i thought they were one and the same, guess i am wrong about that.
> 
> Does that mean i need a 3'd party app to create device-local albums that i can then upload? Or is there maybe a plugin that gives immich access to google pixel?

You can Google takeout your photos & albums from Google Photos and import them easily to Immich with[ Immich go](https://github.com/simulot/immich-go)

**Comment by Sander-Brilman** [comment]
Thank you both for the help. Sorry for reporting it as a bug, I did not know this was a google issue rather then a immich issue but you learn something new every day.

I've managed to solve my issue by using a 3d party image gallery app that creates albums on-device instead of whatever google photos does so immich can now detect it. Although it would be nice if in the future backing up albums from google photo's is an option (if it is even possible in the first place)

nonetheless i enjoy using immich and appricate the help
